lib/tasks/find_possible_duplicate_accounts.rake

```diff
@@ -0,0 +1,64 @@
+namespace :accounts do
+  namespace :duplicate_accounts do
+    desc "Find possible duplicate acccounts"
+    task :find => [:environment] do
+      CSV.open("duplicate_users_by_name.csv", 'w+') do |csv|
+        csv << ['User First Name', 'User Last Name', 'Email Address(es)', 'User ID', 'Social Signup Successful', 'Reset Password Help Requested', 'Help Request Failed']
```

Comment:
For boolean-like fields, perhaps adding a `?` will make the title more clear (e.g., "Social Signup Successful?")